pkg/model/test.go

```diff
@@ -22,6 +27,31 @@ type Test struct {
 	Caches    []string      `yaml:"caches,omitempty"`
 }
 
+var (
+	// TODO: add link to docs when available
+	errNoTestsDefined = fmt.Errorf("there are no tests configured in your Okteto Manifest")
+)
+
+func (test ManifestTests) Validate() error {
+	if test == nil {
+		return errNoTestsDefined
+	}
+
+	hasAtLeastOne := false
+	for _, t := range test {
+		if t != nil {
+			hasAtLeastOne = true
```

Comment:
I think this code could be simplified by removing the bool and returning nil as soon as we detect there is one without the break. WDYT?
```suggestion
	for _, t := range test {
		if t != nil {
			return nil
		}
	}
```

Reply:
@jLopezbarb I personally find more readable and easier to maintain the current code, because having `return nil` means we cannot add additional validations after this check. If we wanrt to add one, we need to refactor and add the bool. Your suggestion uses less memory but in our context, I think the benefits are minimal compared to keeping the code easy to change. WDYT?